packages/mobile/src/screens/edit-playlist-screen/EditPlaylistFormScreen.tsx

```diff
@@ -0,0 +1,95 @@
+import type { ReactElement, ReactNode } from 'react'
+
+import { Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+
+import type { ScreenProps } from 'app/components/core'
+import { ScreenContent, Button, Screen } from 'app/components/core'
+import { useNavigation } from 'app/hooks/useNavigation'
+import { getIsKeyboardOpen } from 'app/store/keyboard/selectors'
+import { makeStyles } from 'app/styles'
+
+const messages = {
+  cancel: 'Cancel',
+  save: 'Save'
+}
+
+type FormScreenProps = Omit<ScreenProps, 'children'> & {
+  bottomSection?: ReactNode
+  topSection?: ReactNode
+  onReset: () => void
+  onSubmit: () => void
+  goBackOnSubmit: boolean
+  children: ReactElement
+}
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  root: { justifyContent: 'space-between' },
+  buttonContainer: {
+    flexDirection: 'row',
+    gap: spacing(4)
+  },
+  bottomButton: {
+    flexGrow: 1
+  },
+  topSection: {},
+  bottomSection: {
+    padding: spacing(4),
+    paddingBottom: spacing(12),
+    backgroundColor: palette.white,
+    borderTopWidth: 1,
+    borderTopColor: palette.neutralLight7
+  }
+}))
+
+export const FormScreen = (props: FormScreenProps) => {
```

Comment:
so iirc there already is a "FormScreen" component out there that we can maybe use here?

Reply:
oh this is actually not supposed to be here haha. I made this first and then scrapped it